Beams/Logic/vtkSlicerIECTransformLogic.h

```diff
@@ -62,8 +91,13 @@ class VTK_SLICER_BEAMS_LOGIC_EXPORT vtkSlicerIECTransformLogic : public vtkMRMLA
     TableTopEccentricRotation,
     TableTop,
     FlatPanel,
+    WedgeFilter,
+    Patient, // RAS

```

Comment:
Could you add a bit more details here in a comment how IEC Patient, DICOM patient (LPS) and Slicer patient (RAS) coordinate systems are related?

Reply:
I've added additional description